.github/labeler.yml

```diff
@@ -1,3 +1,10 @@
-# Add needs triage label to all new issues
-needs-triage:
+# Add a 'needs triage' label for each browser to all new issues
```

Comment:
```suggestion
# Add a 'needs triage' label for each browser engine to all new issues
```

Let's emphasize that it is a distinct engine. @mukul-p just confirming - this is what we decided, right? In practice Edge hasn't made significant use of the consensus labels in the group, since Edge mostly follows Chromium in its extension implementation. Thus, it would reduce overhead to not require Edge to triage issues too.

Reply:
sgtm.